ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ActivePromoCodeComposable.kt

```diff
@@ -0,0 +1,141 @@
+package com.appcoins.wallet.ui.widgets
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.zIndex
+import com.appcoins.wallet.ui.common.theme.WalletColors
+
+
+@Preview
+@Composable
+private fun CardActivePromoCodeExample() {
+  ActivePromoCodeComposable(cardItem = promoCodeItem)
+}
+
+@Composable
+fun ActivePromoCodeComposable(cardItem: ActiveCardPromoCodeItem) {
+  Column(modifier = Modifier
+    .padding(
+      start = 16.dp,
+      end = 16.dp,
+      top = 18.dp
+    )) {
+    Box(
+      modifier = Modifier
+        .align(Alignment.End)
+        .clip(RoundedCornerShape(topEnd = 8.dp, topStart = 8.dp))
+        .background(WalletColors.styleguide_pink)
+    ) {
+      Text(
+        //Need string to Carlos Translator
+        text = stringResource(id = R.string.promo_code_settings_title),
+        fontSize = 12.sp,
+        color = WalletColors.styleguide_light_grey,
+        modifier = Modifier.padding(
+          top = 6.dp,
+          end = 14.dp,
+          start = 14.dp,
+          bottom = 6.dp
+        )
+      )
+    }
+    Surface(
+      color = WalletColors.styleguide_blue_secondary,
+      modifier = Modifier
+        .border(
+          border = BorderStroke(2.dp,  WalletColors.styleguide_pink),
+          shape = RoundedCornerShape(
+            bottomEnd = 16.dp,
+            bottomStart = 16.dp,
+            topStart = 16.dp
+          )
+        )
+        .clip(
+          shape = RoundedCornerShape(
+            bottomEnd = 16.dp,
+            bottomStart = 16.dp,
+            topStart = 16.dp
+          )
+        )
+        .zIndex(4f)
+    ) {
+      Column(modifier = Modifier.padding(8.dp, 8.dp, 8.dp, 8.dp)) {
```

Comment:
Optional: In these cases you could just use 
`modifier = Modifier.padding(8.dp)`